lib/pester/version.rb

```diff
@@ -1,3 +1,3 @@
 module Pester
-  VERSION = '0.1.0'
+  VERSION = '0.1.1'
```

Comment:
To be a noodge, this is technically a minor bump, per [semver](http://semver.org/)